enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -108,6 +110,61 @@ <h3 class="course-confirmation-title">${ confirmation_text }</h3>
         </div>
       </div>
     </div>
-
+    <div id="course-details-modal" class="modal" aria-hidden="true">
+      <div id="course-details-modal-content" class="modal-content" role="dialog" aria-modal="true" aria-labelledby="modal-header-text">
+        <button class="modal-close-button" id="modal-close-button" aria-label="${_('Close')}">${_('Close')}</button>
+        <header class="wrapper">
+          <div class="image">
+            <img src="${ course_image_uri }" alt="${ course_name }"/>
+          </div>
+          <div class="details">
+            <h1 id="modal-header-text">${ course_name }</h1>
+            <p class="short-description">${ course_short_description }</p>
+            <div class="organization">
+              <img src="${ organization_logo }" alt="${ organization_name }" />
+            </div>
+          </div>
+        </header>
+        <div class="body">
+          <div class="details">
+            <ol>
+              <li>
+                <span class="icon fa fa-tag" aria-hidden="true"></span>
+                <span class="title">${ price_text}</span>
+                <span class="text">
+                  % if course_modes:
```

Comment:
@replaceafill The code below accesses `course_modes[0]`, `course_modes[0]['final_price']`, and `course_modes[0]['original_price']` several times, so it would make sense to introduce variables that reference these values. (See http://docs.makotemplates.org/en/latest/syntax.html#python-blocks for an example.)